src/utils/account.ts

```diff
@@ -1,43 +1,64 @@
 import { Keypair, Mnemonic, Address } from '@helium/crypto-react-native'
 import * as SecureStore from 'expo-secure-store'
 
+type AccountStoreKey = BooleanKey | StringKey
+
+const stringKeys = ['mnemonic', 'keypair', 'address', 'userPin'] as const
+type StringKey = typeof stringKeys[number]
+
+const boolKeys = [
+  'accountBackedUp',
+  'isEducated',
+  'isSettingUpHotspot',
+] as const
+type BooleanKey = typeof boolKeys[number]
+
+export const setItem = async (key: AccountStoreKey, val: string | boolean) => {
```

Comment:
we may want to avoid exporting the arbitrary `setItem` function since that could encourage yolo setting/getting elsewhere in the app. it may be better to have dedicated functions within here

Reply:
It's type checked, so you can't just go nuts. I like it this way because it avoids writing a ton of boilerplate functions. I could be convinced otherwise though.

<img width="209" alt="Image 2020-11-17 at 4 22 06 PM" src="https://user-images.githubusercontent.com/1760821/99457736-120cf300-28f1-11eb-8821-4cc6a0df5c24.png">

Reply:
that should be ok then, just want to make sure we account for the keys we're setting, but looks like this accomplishes that